src/app/backend/replicasetdetail.go

```diff
@@ -73,13 +81,33 @@ func GetReplicaSetDetail(client *client.Client, namespace string, name string) (
 		return nil, err
 	}
 
+	services, err := client.Services(namespace).List(labels.Everything(), fields.Everything())
+
+	if err != nil {
+		return nil, err
+	}
+
+	matchingServices := getMatchingServices(services.Items, replicaSet)
```

Comment:
I mean, you can create a function that returns list of endpoints for service list. And then use this method in two places.